docs/en/stack/ml/nlp/ml-nlp-ner-example.asciidoc

```diff
@@ -0,0 +1,296 @@
+[[ml-nlp-ner-example]]
+= How to deploy named entity recognition (NER)
+
+++++
+<titleabbrev>Named entity recognition</titleabbrev>
+++++
+:keywords: {ml-init}, {stack}, {nlp}
+
+This page demonstrates how to deploy a 
+<<ml-nlp-ner,named entity recognition (NER)>> model in {es}, test the model, and 
+add it to an {infer} ingest pipeline. The model that is used in the example is 
+publically available on https://huggingface.co/[HuggingFace].
+
+
+[discrete]
+[[ex-ner-requirements]]
+== Requirements
+
+include::ml-nlp-shared.asciidoc[tag=nlp-requirements]
+
+
+[discrete]
+[[ex-ner-deploy]]
+== Deploying a NER model
+
+include::ml-nlp-shared.asciidoc[tag=nlp-eland-clone-docker-build]
+
+Select a NER model from the 
+{ml-docs}/ml-nlp-model-ref.html#ml-nlp-model-ref-ner[third-party model reference list].
+This example uses an 
+https://huggingface.co/elastic/distilbert-base-uncased-finetuned-conll03-english[uncased NER model].
+
+Install the model by running the `eland_import_model_hub` command in the Docker 
+image:
+
+[source,shell]
+--------------------------------------------------
+docker run -it --rm elastic/eland \
+    eland_import_hub_model \
+      --cloud-id $CLOUD_ID \
+      -u <username> -p <password> \
+      --hub-model-id elastic/distilbert-base-uncased-finetuned-conll03-english \
+      --task-type ner \
+      --start
+
+--------------------------------------------------
+
+You need to provide an administrator username and its password and replace the 
+`$CLOUD_ID` with the ID of your Cloud deployment. This Cloud ID can be copied 
+from the deployments page on your Cloud website.
+
+include::ml-nlp-shared.asciidoc[tag=nlp-start]
+
+include::ml-nlp-shared.asciidoc[tag=nlp-sync]
+
+
+[discrete]
+[[ex-ner-test]]
+== Testing the NER model
+
+Deployed models can be evaluated using the 
```

Comment:
> Deployed models can be evaluated using the 

Perhaps it's also worth mentioning the Kibana test model route since it's simpler IMO